packages/@aws-cdk/aws-location-alpha/lib/util.ts

```diff
@@ -0,0 +1,9 @@
+import { Names } from 'aws-cdk-lib/core';
+
+export function generateUniqueId(context: any): string {
```

Comment:
Instead of `any`, I believe we can use `IConstruct`

Reply:
Thanks.
I've updated.